vars/piperPipelineStageInit.groovy

```diff
@@ -171,7 +172,16 @@ void call(Map parameters = [:]) {
             .addIfEmpty('buildTool', script.commonPipelineEnvironment.buildTool)
             .withMandatoryProperty('buildTool')
             .use()
-
+        
+        echo "YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY"
```

Comment:
why is this echo? May be you need to clean it up?

Reply:
cleaned as suggested